cmd/goal/README.md

```diff
@@ -87,82 +92,25 @@ goal account balance --address ${APP_ACCOUNT}
-* the app id has been stored in `$APPID` (see the previous answer)
+* the app id has been stored in `${APPID}` (see the previous answer)
 * the box referenced in the first app-call has name `greatBox`
-* another referenced box has abi-name of type `(byte,byte,byte)` and name `[2,3,5]`
-* this second box is provided contents `42`
-* `msgpacktool` is used to decode to JSON
-* it is assumed that `$ALGORAND_DATA` has been exported (see previous answer)
+* another referenced box is named `[2,3,5]` of ABI-type `(byte,byte,byte)`
+* this second box is provided contents `fourty two`
 
 ```sh
-# create a box with a simple non-ABI name:
-❯ goal app call --from $ACCOUNT --app-id ${APPID} --app-arg "str:create" --app-arg "str:greatBox" -o - | msgpacktool -d
-{
-  "txn": {
-    "apaa": [
-      "Y3JlYXRl",
-      "Z3JlYXRCb3g="
-    ],
-    "apid": 1,
-    "fee": 1000,
-    "fv": 314,
-    "gh:b64": "C6TkcPi8SaIgArGPLnbqRpQ+obwjMmgGnLJ+X0hQkFE=",
-    "lv": 1314,
-    "note:b64": "gd5dwCVwxpY=",
-    "snd:b64": "GKgl/BbeLf7kecfIA8RKWlzr/p44bYt5tLtRsvt39Bc=",
-    "type": "appl"
-  }
-}
-
-# create a box named by an ABI-type:
-❯ goal app call --from ${ACCOUNT} --app-id ${APPID} --app-arg "str:create" --app-arg "abi:(byte,byte,byte):[2,3,5]" -o - | msgpacktool -d
-{
-  "txn": {
-    "apaa": [
-      "Y3JlYXRl",
-      "AgMF"
-    ],
-    "apid": 1,
-    "fee": 1000,
-    "fv": 589,
-    "gh:b64": "ETw2drwLuKVe1YKOszAZVG4262GYFWo3Sv9FaPaxKyQ=",
-    "lv": 1589,
-    "note:b64": "mUpBGHTlBrI=",
-    "snd:b64": "/vIHevqRpLcRyZkYoUr6jLBdGZdmfOUaVDuiNEIuz9g=",
-    "type": "appl"
-  }
-}
-
-# set the ABI-type box name contents. Note how the `--box` flag needs to be set so as to refer to the box being touched:
-❯ goal app call --from ${ACCOUNT} --app-id ${APPID} --app-arg "str:set" --app-arg "abi:(byte,byte,byte):[2,3,5]" --app-arg "str:42" --box "${APPID},abi:(byte,byte,byte):[2,3,5]" -o - | msgpacktool -d
-{
-  "txn": {
-    "apaa": [
-      "c2V0",
-      "AgMF",
-      "NDI="
-    ],
-    "apbx": [
-      {
-        "n:b64": "AgMF"
-      }
-    ],
-    "apid": 1,
-    "fee": 1000,
-    "fv": 413,
-    "gh:b64": "C6TkcPi8SaIgArGPLnbqRpQ+obwjMmgGnLJ+X0hQkFE=",
-    "lv": 1413,
-    "note:b64": "h/N9VcxOBpw=",
-    "snd:b64": "GKgl/BbeLf7kecfIA8RKWlzr/p44bYt5tLtRsvt39Bc=",
-    "type": "appl"
-  }
-}
+# create a box with a simple non-ABI name. Note how the `--box` flag needs to be set so as to refer to the box being touched
+goal app call --from $ACCOUNT --app-id ${APPID} --box "str:greatBox" --app-arg "str:create" --app-arg "str:greatBox"
+
+# create a box named by an ABI-type
+goal app call --from ${ACCOUNT} --app-id ${APPID} --box "abi:(byte,byte,byte):[2,3,5]" --app-arg "str:create" --app-arg "abi:(byte,byte,byte):[2,3,5]"
+
+# set the ABI-type box name contents
+goal app call --from ${ACCOUNT} --app-id ${APPID} --box "${APPID},abi:(byte,byte,byte):[2,3,5]" --app-arg "str:set" --app-arg "abi:(byte,byte,byte):[2,3,5]" --app-arg "str:fourty two"
```

Comment:
I think this is going to lead to some confusion.  I think the reader is going to imagine that it shows how to store a `[2,3,5]` tuple for later use.  But it's using the encoding of a `[2,3,5]` tuple as a _name_ and storing the bytes "fourty two" as the value.  I think a more relatable example would be to create a box _named_ something like "scores", and then use the abi encoding technique to store a tuple like '[95,82,75]' in there.